app/routes/bookmarks/new.tsx

```diff
@@ -0,0 +1,20 @@
+import type { ActionFunction } from "@remix-run/node";
+import { redirect } from "@remix-run/node";
+import { bookmarkService } from "~/features/bookmarks/bookmark.service.server";
+import { BookmarkForm } from "~/features/bookmarks/BookmarkForm";
+import { AppErrorBoundary } from "~/features/error/AppErrorBoundary";
+
+export default function NewBookmarkRoute() {
+  return <BookmarkForm />;
+}
+
+export const action: ActionFunction = async ({ request }) => {
```

Comment:
Is there value in sharing an action between new and edit? same way we share a data service method, `save()` ?